central/platform/reprocessor/reprocessor_impl.go

```diff
@@ -65,20 +76,50 @@ func (pr *platformReprocessorImpl) Stop() {
 	pr.stopSignal.Signal()
 }
 
-func (pr *platformReprocessorImpl) runReprocessing() {
-	err := pr.reprocessAlerts()
+func (pr *platformReprocessorImpl) RunReprocessor() {
+	log.Info("Running platform reprocessor")
+	err := pr.semaphore.Acquire(reprocessorCtx, 1)
 	if err != nil {
-		log.Errorf("Error reprocessing alerts with platform rules: %v", err)
+		log.Errorf("Failed to acquire reprocessor semaphore: %v", err)
+		return
+	}
+	flag := true
+	if pr.customized {
+		log.Info("Customized platform reprocessor")
+		config, _, err := pr.configDatastore.GetPlatformComponentConfig(reprocessorCtx)
+		if err != nil {
+			log.Errorf("Error getting platform component config config: %v", err)
+		}
+		flag = config.NeedsReevaluation
 	}
+	log.Infof("Reprocessor started, flag: %v", flag)
```

Comment:
`[Platform Reprocessor] Reprocessor started`, this will help differentiate it from other kinds of reprocessors.